crates/biome_rule_options/src/use_unified_type_signatures.rs

```diff
@@ -1,6 +1,18 @@
 use biome_deserialize_macros::{Deserializable, Merge};
 use serde::{Deserialize, Serialize};
-#[derive(Default, Clone, Debug, Deserialize, Deserializable, Merge, Eq, PartialEq, Serialize)]
+
+
+#[derive(
+    Clone, Copy, Debug, Default, Deserialize, Deserializable, Merge, Eq, PartialEq, Serialize,
+)]
 #[cfg_attr(feature = "schema", derive(schemars::JsonSchema))]
 #[serde(rename_all = "camelCase", deny_unknown_fields, default)]
-pub struct UseUnifiedTypeSignaturesOptions {}
+pub struct UseUnifiedTypeSignaturesOptions {
+    /// Whether to ignore overloads with differently named parameters.
+    #[serde(skip_serializing_if = "Option::<_>::is_none")]
+    pub ignore_differently_named_parameters: Option<bool>,
+
+    /// Whether to ignore overloads with different JSDoc comments.
+    #[serde(skip_serializing_if = "Option::<_>::is_none")]
+    pub ignore_different_jsdoc: Option<bool>,
+}
```

Comment:
nit: Consider adding an impl block for these so you don't have to write `unwrap_or_default()` when using them.

```rust
impl UseUnifiedTypeSignaturesOptions {
  fn ignore_differently_named_parameters() -> bool { ... }
  fn ignore_different_jsdoc() -> bool { ... }
}
```
Plus, it makes it way less error prone to change the defaults if we decide to do that later.